h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -264,6 +265,20 @@ public boolean onExceptionalCompletion(Throwable ex, CountedCompleter caller) {
     public abstract void computeImpl();
   }
 
+  public void assertEffectiveParameters(Model.Parameters effectiveParameters){
+    try {
+      for (Field field : effectiveParameters.getClass().getFields()) {
+        Class type = field.getType();
+        Object value = field.get(effectiveParameters);
+        if (value != null/* && "AUTO".equals(value.toString())*/){
+          assert(!"AUTO".equals(value.toString())) : "FAIL: found AUTO value in effective parameters: " + field.getName();
+        }
+      }
+    } catch (IllegalAccessException e) {
+      e.printStackTrace();
```

Comment:
Params should have all fields public/readable so this should not happen so rethrowing as RuntimeException should be fine

Reply:
fixed